quotefaultAPI/__init__.py

```diff
@@ -79,11 +79,7 @@ def between(start: str, limit: str, api_key: str):
     :return: Returns a JSON list of quotes between the two dates
     """
     if check_key(api_key):
-        if datetime.strptime(start, "%Y-%m-%d") < datetime.strptime(limit, "%Y-%m-%d"):
-            quotes = Quote.query.filter(Quote.quoteTime.between(start, limit)).all()
-            return jsonify(parse_as_json(quotes))
-        quotes = Quote.query.all
-        return jsonify(parse_as_json(quotes))
+		return jsonify(parse_as_json(query_builder(start, limit, request.args.get('submitter').all()))
```

Comment:
This is getting to be a deeply nested line. Try and break it up for the sake of readability, whether in a function or just into variable storage